index.html

```diff
@@ -59,5 +59,170 @@ <h3>LET'S CONNECT</h3>
         </ul>
       </section>
     </main>
+    <!-- Cards section -->
+    <section class="main-container-projects">
+      <div class="cards card-1">
+        <div class="main-img">
+          <img src="./img/Snapshoot Portfolio.png" alt="person" />
+        </div>
+        <article class="second-container">
+          <h2>Tonic</h2>
+          <div class="horizontal-section">
+            <span>CANOPY</span>
```

Comment:
- [Optional] Please fix the position of the background image it is up a little bit 
![image](https://user-images.githubusercontent.com/74744787/219783171-500369f3-f38f-4280-ba50-9ead67cbfc62.png)